terraform/lib/dependabot/terraform/file_parser.rb

```diff
@@ -241,7 +241,7 @@ def git_source_details_from(source_string)
           if git_url.include?("git@")
             git_url.split("git@").last.sub(":", "/")
           else
-            git_url.sub(%r{.*?://}, "")
+            git_url.sub(%r{(?:\w{3,5})?://}, "")
```

Comment:
This fixed the CodeQL issue that was flagged. All protocols git allowes are betwen 3 and 5 characters: `git`, `ssh`, `http`, `https`, `file`.